docs/ops/normalization/GroupNormalization_12.md

```diff
@@ -0,0 +1,79 @@
+# GroupNormalization {#openvino_docs_ops_normalization_GroupNormalization_12}
+
+@sphinxdirective
+
+**Versioned name**: *GroupNormalization-12*
+
+**Category**: *Normalization*
+
+**Short description**: Performs normalization of the input tensor according to the method described in https://arxiv.org/abs/1803.08494
+
+**Detailed description**
+
+The GroupNormalization operation performs the following transformation of the input tensor:
+
+  `y = scale * (x - mean) / sqrt(variance + epsilon) + bias`
+
+The operation is applied per batch, per group of channels. This means that the example input with `N x C x H x W` layout is transformed to the `N x G x C/G x H x W` form. The `scale` and `bias` coefficients are the inputs to the model and need to be specified separately for each group. The `mean` and `variance` are calculated for each group and applied accordingly.
+
+**Attributes**
+
+* *num_groups*
+
+  * **Description**: Specifies the number of groups `G` that the channel dimesion will be divided into.
+  * **Range of values**: between `1` and the number of channels `C` in the input tensor
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *epsilon*
+
+  * **Description**: A very small value added to the variance for numerical stability. Ensures that division by zero does not occur for any normalized element.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Required**: *yes*
+
+**Inputs**
+
+* **1**: `data` - The input tensor to be normalized. The type of this tensor is *T*. The tensor's shape is arbitrary but the first two dimensions are interpreted as `batch` and `channels` respectively. **Required.**
+
+* **2**: `scale` - 1D tensor of type *T* containing the scale values for each group. The expected shape of this tensor is `[G]` where `G` is equal to the value of the `num_groups` attribute. **Required.**
+
+* **3**: `bias` - 1D tensor of type *T* containing the bias values for each group. The expected shape of this tensor is `[G]` where `G` is equal to the value of the `num_groups` attribute. **Required.**
```

Comment:
In the paper or Pytorch docs the `scale`(gamma) and `bias`(beta) are described as learnable per-channel.
Based on that I would expect that the shape of those inputs will be related to the channels [C] instead of number of groups [G]. 
I understand that [G] is aligned with ONNX, but is there any reason of such inconsistency?

Reply:
I've done more research and I think ONNX's documentation is incorrect, it should expect the equality with channel dimension(like in the paper) and not with `G`. I've changed it in the spec.